mesonbuild/interpreter/interpreter.py

```diff
@@ -2668,6 +2675,12 @@ def check_clang_asan_lundef(self) -> None:
     # subproject than it is defined in (due to e.g. a
     # declare_dependency).
     def validate_within_subproject(self, subdir, fname):
+        if isinstance(fname, P_OBJ.DependencyVariableString):
+            norm = Path(fname)
+            # variables built from a dep.get_variable are allowed to refer to
+            # subproject files. FIXME: check if they are installed?
```

Comment:
Note: I'm wondering what to do for this FIXME. I'm also wondering whether I *should* do this FIXME.